charts/sourcegraph/files/codeinsights-db/conf/postgresql.conf

```diff
@@ -0,0 +1,57 @@
+#------------------------------------------------------------------------------
+# POSTGRESQL DEFAULT CONFIGURATION
+#------------------------------------------------------------------------------
+
+# Below is PostgreSQL default configuration.
+# You should apply your own customization in the CUSTOMIZED OPTIONS section below
+# to avoid merge conflict in the future.
+
+listen_addresses = '*'
+max_connections = 100
+shared_buffers = 128MB
+dynamic_shared_memory_type = posix
+max_wal_size = 1GB
+min_wal_size = 80MB
+log_timezone = 'UTC'
+datestyle = 'iso, mdy'
+timezone = 'UTC'
+lc_messages = 'en_US.utf8'
+lc_monetary = 'en_US.utf8'
+lc_numeric = 'en_US.utf8'
+lc_time = 'en_US.utf8'
+default_text_search_config = 'pg_catalog.english'
+
+
+#------------------------------------------------------------------------------
+# SOURCEGRAPH RECOMMENDED OPTIONS
+#------------------------------------------------------------------------------
+
+# Below is Sourcegraph recommended Postgres configuration based on the default resource configuration.
+# You should apply your own customization in the CUSTOMIZED OPTIONS section below
+# to avoid merge conflict in the future.
+
+shared_buffers = 509546kB
+work_mem = 3184kB
+maintenance_work_mem = 254773kB
+effective_io_concurrency = 200
+max_worker_processes = 19
+max_parallel_workers_per_gather = 4
+max_parallel_workers = 8
+wal_buffers = 15285kB
+min_wal_size = 512MB
+checkpoint_completion_target = 0.9
+random_page_cost = 1.1
+effective_cache_size = 1492MB
+default_statistics_target = 500
+autovacuum_max_workers = 10
+autovacuum_naptime = 10
+shared_preload_libraries = ''
+max_locks_per_transaction = 64
+
+
+#------------------------------------------------------------------------------
+# CUSTOMIZED OPTIONS
+#------------------------------------------------------------------------------
+
+# Add your customization here
```

Comment:
Is this instruction (and the two other references above) confusing now in helm? Customers won't be adding the customizations directly there, but by setting the additionalConfig value.